src/rp2040.ts

```diff
@@ -48,7 +49,8 @@ export class RP2040 {
   readonly usbDPRAM = new Uint8Array(4 * KB);
   readonly usbDPRAMView = new DataView(this.usbDPRAM.buffer);
 
-  readonly core = new CortexM0Core(this);
+  readonly core0 = new CortexM0Core(this);
```

Comment:
Why not using an array/list for the cores? Seems that would make quite a few places of the code simpler, and `RP4040` might be just around the corner anyways. ;)

Reply:
At the beginning, I use array of cores, but found so may cases need to handle for each core(like notify another core interrupt or operations no SIO) which makes the benefit of array less and make the code not easy to read(compare to use 2 variable). It's just my opionion.